extensions/ConfirmEdit/ReCaptcha.class.php

```diff
@@ -73,13 +64,13 @@ function addCaptchaAPI( &$resultArr ) {
 	 * @param $action Action being performed
 	 * @return string
 	 */
-	function getMessage( $action ) {
+	public function getMessage( $action ) {
+		// Possible keys for easy grepping: recaptcha-edit, recaptcha-addurl, recaptcha-createaccount, recaptcha-create
```

Comment:
Wonderful comment! :+1: With such a small effort put to place it here it'll be so useful in future :)